fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -549,6 +564,21 @@ public Resource insert(Resource resource, List<ExtractedParameterValue> paramete
 
             resource.setId(stmt.getLong(7));
 
+            if (large) {
+                // Use the long id to update the record in the database.
+                String largeStmtString = String.format(LARGE_BLOB, getSchemaName(), resource.getResourceType());
+                try (PreparedStatement ps = connection.prepareStatement(largeStmtString);) {
```

Comment:
; is redundant

Reply:
Yeah - sometimes I get in the habit, as I often add second closeables, I've removed in this case